src/Components/Blinds/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import "./style.css";
+import blindsOpen from "./images/blinds-open.svg"
+import blindsClosed from "./images/blinds-closed.svg"
+
+const Blinds = ({ state }) => {
+    const [blindsState, setBlindsState] = useState(state)
```

Comment:
(jako u Light by se tu dal použít boolean kdybys chtěla, ale zase, není to špatně, je to jen otázka designu autora)